src/m3u8/m3u8-parser.js

```diff
@@ -436,7 +440,8 @@
               // setup an encryption key for upcoming segments
               key = {
                 method: entry.attributes.METHOD || 'AES-128',
-                uri: entry.attributes.URI
+                uri: entry.attributes.URI,
+                iv: entry.attributes.IV || null
```

Comment:
@rajkosto this is a great place for the key when it's present in the manifest. If it's not present though, we've been signaling that by not including the property name. Could you update this so that `entry.attributes.IV` is only set if a custom IV is present in the m3u8?